modules/mux/targets/host/source/device.cpp

```diff
@@ -432,10 +432,10 @@ device_info_s::device_info_s(host::arch arch, host::os os, bool native,
   this->clock_frequency = native ? os_cpu_frequency() : 0;
   this->compute_units = native ? os_num_cpus() : 0;
   this->buffer_alignment = sizeof(uint64_t) * 16;
-  // TODO redmine(8386): Reported memory size is quartered in order to pass the
-  // OpenCL CTS however this probably should be an OCL specific fix and not in
-  // the host target.
-  this->memory_size = native ? os_memory_bounded_size() / 4 : 0;
+  // TODO redmine(8386): Reported memory size is quartered (rounded up) in order
```

Comment:
might be worth checking the redmine to see if this still has relevance.

Reply:
Checked, it still has relevance, but it's in a place where we would lose track of it so I created a new internal ticket so that we do keep track of it.